table.rst

```diff
@@ -0,0 +1,45 @@
+
+Table for astype
+
+
++-------------------+-------------------------+
+|  S1               |  Dtype                  |
++===================+=========================+
+|  int              | pd.Int8Dtype,           |
+|                   | pd.Int16Dtype,          |
+|                   | pd.Int32Dtype,          |
+|                   | pd.Int54Dtype,          |
+|                   | "int",int,np.int8,      |
+|                   |  np.int16,np.int32,     |
+|                   |  np.int64,np.uint8      |
+|                   |  np.uint16,np.uint32    |
+|                   |  np.uint64,np.intp      |
+|                   |  np.uintp,np.integer    |
++-------------------+-------------------------+
+|  str              | StringDtype,str,"str"   |
++-------------------+-------------------------+
+|  bytes            | np.byte,np.ubyte,bytes  |
+|                   | "bytes"                 | 
++-------------------+-------------------------+
+|  float            | pd.Float32Dtype,        |
+|                   | pd.Float64Dtype,        |
+|                   | float,"float",np.float16|
+|                   | np.float32,np.float64   |
+|                   | np.float96,np.float128  |
+|                   | np.floating             |
++-------------------+-------------------------+
+|  complex          | "complex",np.complex64, |
+|                   | np.complex128,          |
+|                   | np.complex192,          |
+|                   | np.complex256,          |
+|                   | complex                 |
++-------------------+-------------------------+
+|  bool             | pd.BooleanDtype, bool   |
+|                   | np.bool_                |
+|                   | "bool"                  |
++-------------------+-------------------------+
+| np.timedelta64    | np.timedelta64          |
++-------------------+-------------------------+
+|  np.datetime64    | np.datetime64           |
```

Comment:
last two rows need to include the appropriate `datetime` types.

If you do that, I think the table is then correct.

Reply:
Also need to include the string representations like `"datetime64[ns]"` that convert to the numpy representations.

Reply:
> Also need to include the string representations like `"datetime64[ns]"` that convert to the numpy representations.

yes

Reply:
i added "datetime64[ns]", i don't think there are any more datimetime types

Reply:
Well, it turns out that `np.timedelta64` and `np.datetime64` should not be in `S1`, so they are now removed.  However `Timestamp` and `Timedelta` are there.  For `Timestamp`, aside from `datetime64[ns]`, additional resolutions will be supported in the future, but we can worry about that in the future.  So can you make sure that all the current types for `S1` are in your table?

Reply:
yes, i'll do it